docs/source/getting_started/content/proc_configuring-the-virtual-machine.adoc

```diff
@@ -0,0 +1,50 @@
+[id="configuring-the-virtual-machine_{context}"]
+= Configuring the virtual machine
+
+The default number of vCPUs and amount of memory available to the {prod} virtual machine can be configured using the `cpus` and `memory` configurable properties, respectively.
+
+[IMPORTANT]
+====
+Changes to the configuration of the virtual machine require the creation of a new {prod} virtual machine.
```

Comment:
It sounds like I can't actually change the configuration of my current vm. Is that the case?
Are you saying that if I change the configuration of the vm, I have to create a new vm for those changes to take effect? If so, what happens to my existing vm? Do I need to destroy it?

Reply:
At the moment we can not modify a VM in-place due to a limitation in the libmachine library we use.

Yes, a new VM is needed.

Reply:
OK, that's not clear to me when I read this.

Reply:
The following would be clearer to me:

You cannot modify the configuration of an existing virtual machine. To enable configuration changes, you need delete the current vm and create a new vm.

Reply:
We hope to resolve this in the future... but for now a caution note might be needed as this is destructive.

Reply:
@gbraad I'm not sure what you're referring to when you say "this is destructive". A caution  is fine. I was just commenting on what I understood from the text.